### PR TITLE
Portfolio and mobile version- skeleton

### PR DESCRIPTION
**Pull request by kelomo2502**
**In this first milestone :**

- Set up a new repository and prepare it for development using best practices (e.g. linters)
- Choose a template from Figma and parse to create a user interface
- Build the first 2 sections of the mobile website using the template chosen from figma
- Build the first two sections of the mobile portfolio using flexbox

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
     steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
-
-      # Runs a single command using the runners shell
-      - name: Run a one-line script
-        run: echo Hello, world!
-
-      # Runs a set of commands using the runners shell
-      - name: Run a multi-line script
```
Comment (Whoistolu) [inline .github/workflows/linters.yml:33]: - Kindly note that you are not allowed to alter the `linter` files. There should be no custom changes to `linter` config.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Fira+Sans:wght@400;700&display=swap" rel="stylesheet">
+
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+      <nav class="nav-bar">
+            <a href="#">Gbenga</a>
+            <img class="logo" src="./images/hamburger/Union.png" alt="drop-menu">   
+      </nav>
```
Comment (Whoistolu) [inline index.html:18]: - [ ] [REQUIRED] Kindly note that the container of the navigation bar should be a `header` tag. The `header` tag in HTML is used to define the header for a document or a section as it contains the information related to the title and heading of the related content. The `header` element is intended to usually contain the section's heading. Kindly implement this.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
+
+/* CSS rules to specify families
+
+font-family: 'Fira Sans', sans-serif; */
```
Comment (Whoistolu) [inline style.css:4]: - Kindly remove all unused lines of code in order to have a clean code-base.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+                </div>
+
+                <ul class="social-icons">
+                   <li class="three"><img src="./images/git.png" alt="vector3"></li>
+                   <li class="four"><img src="./images/group-linkedin.png" alt="vector4"></li>
+                   <li class="six"><img src="./images/vector5.png" alt="vector4"></li>
+                   <li class="seven"><img src="./images/vector6.png" alt="vector6"></li>
+                   <li class="eight"><img src="./images/vector7.png" alt="vector6"></li>
+                </ul>
```
Comment (Whoistolu) [inline index.html:41]: - [ ] [REQUIRED] Great job implementing the social icons in an unordered list (ul) and (li) for each icon, however, not all icons seem to be displaying properly. Kindly fix this in order to meet up with the project requirement.
### Your Icons
Screenshot from 2022-03-21 19-09-54

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @kelomo2502 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kelomo2502** [comment]
Thank you for your email.
Will rework and revert back to you.

Gbenga

File: style.css [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
```
Comment (ShoiraTa) [inline style.css:4]: - [ ] As the previous reviewer requested, please remove all unused lines of code in order to have a clean code-base.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
```
Comment (ShoiraTa) [inline index.html:41]: - [ ] As the previous reviewer requested, please make sure that all icons are being rendered properly as provided in Figma design: 

Your design  | Figma design
------------- | -------------
Screenshot 2022-03-22 at 14 05 11 | Screenshot 2022-03-22 at 14 06 15

Eg.: There is a typo in `/images/vector6.png` it should be `/images/Vector6.png`

- [ ] Please set the color of icons to `#DBD8D7` curretly only Linkedin icon is of similar color.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+<link href="https://fonts.googleapis.com/css2?family=Fira+Sans:wght@400;700&display=swap" rel="stylesheet">
+
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+      <header class="nav-bar">
+            <a href="#">Gbenga</a>
+            <img class="logo" src="./images/hamburger/Union.png" alt="drop-menu">   
+      </header>
+      <section class="headline">
```
Comment (ShoiraTa) [inline index.html:19]: - [ ] As the previous reviewer requested, kindly remove the excess white space at the bottom of the screen and the one on the right so your project can look like the template provided. Currenlty, there is white space if the screen size gets smaller than `392px`

Your design  | Figma design
------------- | -------------
Screenshot 2022-03-22 at 14 12 31  | Screenshot 2022-03-22 at 14 13 14

- [  ]OPTIONAL:  Please make sure to decrease the font size as provided in Figma to `56px`

- [ ] OPTIONAL:  Please make sure to have the same font family as in Figma, which is 'Poppins'

- [ ] After making all the changes, please make sure that your page looks same as the design in Figma  🙏🏻

File: index.html [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+            <div class="supporting-text">
+              <h2>I am a Software Developer</h2>
+            </div>
+            
+                <p class="description">I can help you build  a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+                </p>
+                <div class="side-img">
+                  <img class="one" src="./images/side.png" alt="vector1">
+
```
Comment (ShoiraTa) [inline index.html:32]: - [ ] OPTIONAL:  This image is supposed to be a background image. Therefore, please upload it from css as a background image

File: index.html (hunk 7 above)
Comment (ShoiraTa) [inline index.html:19]: - [ ] OPTIONAL:  Headline section had to be aligned with `flexbox`. Please try to use `flexbox` to align this section

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi @kelomo2502,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost


 ## Required Changes  :recycle: 

_Check the comments under the review._ 

## Optional suggestions

_Check the comments under the review._ 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kelomo2502** [comment]
Screenshot (9)

- All Icons display correctly
- Icons color are set to figma template
- Extra Whites spaces removed
- Background-Image uploaded from style.css

**Review by gemmen29 — APPROVED** [review]
Hi @kelomo2502, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions:
- [ ] Kindly give the branch a meaningful name, not `request-review` like the name of the project `setup-and-mobile-version-skeleton`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**